Readme.md

```diff
@@ -1,29 +1,33 @@
-Minimum supported OS
-=====
+[![macos-catalina](https://img.shields.io/badge/macos-catalina-brightgreen.svg)](https://www.apple.com/macos/catalina-preview)
+[![macos-mojave](https://img.shields.io/badge/macos-mojave-brightgreen.svg)](https://www.apple.com/lae/macos/mojave)
 
-This version of ShockEmu has been tested on:
-* macOS 10.13: High Sierra
-* macOS 10.14: Mojave
+# Key Mapping
+`only_keyboard.se` goes like this:
 
-Setup
-=====
+![Key Mapping](https://github.com/backslash-f/ShockEmu/blob/master/KeyMapping.png)
 
-ShockEmu requires the OS X Command Line Tools to be installed.  
+# Setup
+```zsh
+git clone git@github.com:backslash-f/ShockEmu.git
+cd ShockEmu
+./build.sh only_keyboard.se
+./run.sh
+```
 
-Installation steps:
+It depends on your system having [PS4 Remote Play](https://remoteplay.dl.playstation.net/remoteplay/lang/en/index.html) installed at `/Applications/RemotePlay.app`. If this is not the case, you'll need to modify `run.sh` accordingly.
 
-	git clone https://github.com/maephisto666/ShockEmu.git
-	./build.sh <filename>.se
-	./run.sh
+The `OS X Command Line Tools` needs [to be installed](https://stackoverflow.com/a/53078282/584548).
```

Comment:
update permissions because I had to download it as a zip file, i couldn't use terminal for it